CHANGELOG.md

```diff
@@ -4,6 +4,355 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.19.0](https://github.com/go-gitea/gitea/releases/tag/1.19.0) - 2023-03-19
+
+* BREAKING
+  * Add loading yaml label template files (#22976) (#23232)
+  * Make issue and code search support camel case for Bleve (#22829)
+  * Repositories: by default disable all units except code and pulls on forks (#22541)
+  * Support template for merge message description (#22248)
+  * Fix wrong default value for update checker on app.example.ini (#22084)
+  * Remove ONLY_SHOW_RELEVANT_REPOS setting (#21962)
+  * Implement actions (#21937)
```

Comment:
Why it's a breaking?

Reply:
Added a new reserved user name `gitea-actions`.

Reply:
But it is a big feature indeed. the "reserved user name" is just a side effect.